tests/console/podman.pm

```diff
@@ -33,6 +33,12 @@ use version_utils qw(is_sle is_leap);
 sub run {
     select_console("root-console");
 
+    if (is_sle '>=15') {
+        add_suseconnect_product('sle-module-containers');
+    }
+
+    zypper_call 'in podman';
```

Comment:
This has to be executed conditionally, since microos and kubic must have podman already installed (it is a bug if not installed)

Reply:
Updated the commit to not install on microos.